frontend/src/contracts.ts

```diff
@@ -108,7 +108,10 @@ export function getConfigValue(key: string): any {
 
   if(process.env.NODE_ENV === 'development') return '';
   const env = window.location.href.startsWith('https://test') ? 'test' : 'production';
-  const chain = localStorage.getItem('currentChain') || 'BNB';
+  let chain = localStorage.getItem('currentChain') || 'BNB';
+  if(!config.supportedChains.includes(chain)){
+    chain = 'BNB';
```

Comment:
Should we also maybe set localStorage to a supported chain here?

Reply:
https://github.com/CryptoBlades/cryptoblades/pull/1376/commits/0b5a56061a597206b5a4cdbef739c3d163a4afb1